spec/models/article_spec.rb

```diff
@@ -9,51 +9,19 @@
   end
 
   describe 'with valid article' do
-    let (:article) {
-      Article.new({
-                    title: 'Monkey',
-                    body: 'Chicken',
-                    page: 1,
-                    author_name: 'Mike'
-                  })
-    }
+    let (:article) { build(:article) }
 
     it 'should set defaults' do
-      expect(article.published).to be_falsey
+      expect(article.published).to be_truthy
       expect(article.impressions_count).to eq(0)
     end
   end
 
   describe 'with a published and unpublished article' do
-    after(:each) {
-      Article.destroy_all
-      Page.destroy_all
-    }
     before(:each) {
-      Page.destroy_all
-      Article.destroy_all
-
-      Article.create!({
-                       title: 'Monkey',
-                       body: 'Chicken',
-                       page: 1,
-                       author_name: 'Mike',
-                       published: false
-                      })
-      Article.create!({
-                        title: 'Alligator',
-                        body: 'Chicken',
-                        page: 1,
-                        author_name: 'Jim',
-                        published: true
-                      })
-      Article.create!({
-                        title: 'French Press',
-                        body: 'Chicken',
-                        page: 1,
-                        author_name: 'Megan',
-                        published: true
-                      })
+      create(:article, title: 'Monkey', author_name: 'Mike', published: false)
+      create(:article, title: 'Alligator', author_name: 'Jim', published: true)
+      create(:article, title: 'French Press', author_name: 'Megan', published: true)
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.